code/obj/table.dm

```diff
@@ -309,6 +309,8 @@
 				for (var/mob/N in AIviewers(usr, null))
 					if (N.client)
 						shake_camera(N, 4, 1, 0.5)
+			if(istype(user:mutantrace, /datum/mutantrace/monkey))
```

Comment:
`:`?

Reply:
I don't know why I thought that was acceptable.
Memory tells me I copied it from somewhere, but I can't fine `user:mutantrace` anywhere else in the codebase